src/components/Widget/inputs.tsx

```diff
@@ -21,10 +21,13 @@ export function useSyncWidgetInputs(defaultToken?: Currency) {
   })
 
   useEffect(() => {
-    setTokens({
-      [Field.OUTPUT]: defaultToken,
-    })
-    setAmount(EMPTY_AMOUNT)
+    // Add some hysteresis - if there is no new default, do not update the display.
```

Comment:
nit: prob don't need this comment

Reply:
I'd prefer to keep it to avoid someone trying to clean up the code and deleting the if-statement.

Reply:
we have the usePrevValue hook somewhere for looking for state changes - is that more clear?